bitwarden_license/src/Scim/Utilities/ScimServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Bit.Scim.Queries.Users;
+using Bit.Scim.Queries.Users.Interfaces;
+
+namespace Bit.Scim.Utilities;
+
+public static class ScimServiceCollectionExtensions
+{
+    public static void AddScimCommands(this IServiceCollection services)
```

Comment:
I want to avoid any confusion around queries/commands. Also, looking at how the organization commands/queries are registered, they're grouped by domain/feature. I suggest the following methods here:

* `AddScimUserQueries`
* `AddScimUserCommands`
* `AddScimGroupQueries`
* `AddScimGroupCommands`

(only add what you need in this PR though)

Reply:
I've renamed the methods in all branches.